### PR TITLE
Pull Request for the Feature to include forbidden areas

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: sensor_placement/common/src/particle.cpp [hunk 1]
```diff
@@ -491,22 +515,22 @@ void particle::updateTargetsInfo(size_t sensor_index)
...
     for (uint32_t x = left_index; x < right_index; x++)
-    { 
+    {
 
       // now check every potential target in the sensors' bounding box
       if(targets_with_info_.at(y * map_.info.width + x).potential_target == 1)
       {
         // now we found a target
-        if(!targets_with_info_.at(y * map_.info.width + x).occupied)
+        if(!targets_with_info_.at(y * map_.info.width + x).forbidden)
```
Comment (fmirus) [inline sensor_placement/common/src/particle.cpp]: I think the line from groovy_dev with "occupied" is correct. Please replace "forbidden" with "occupied" to fix this bug.

**Comment by fmirus** [comment]
I think there is still a small bug in particle.cpp: In line 533 "occupied" should be replaced with "forbidden". Also there is still the Visualization_msgs/Markers.h included in sensor_placement_node.h which is not needed. Please fix these minor issues and add another commit to this branch/Pull Request.

**Comment by fmirus** [comment]
including Visualization_msgs/Markers.h included in sensor_placement_node.h is correct. Sorry, my bad ;-)

**Comment by ghost** [comment]
Thank you for pointing that out. I've added a commit with the bug fixed

**Pull request merged**